conda_smithy/templates/meta.yaml.ci-skel.tmpl

```diff
@@ -0,0 +1,71 @@
+{{ '{%' }} set name = "{{ package_name }}" {{ '%}' }}
+{% raw %}{% set version = environ.get('GIT_DESCRIBE_TAG', 'untagged')|string|replace('-','_') %}
+{% set build_number = (environ.get('GIT_DESCRIBE_NUMBER')|int + 1000)|string %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}
```

Comment:
If `GIT_DESCRIBE_TAG` is unset, the build will crash with some kind of stack trace. I think it is fine that the build would fail, indicating that something's missing (no `git init`/`git commit`). With
```suggestion
  version: {{ environ['GIT_DESCRIBE_TAG'] }}
```
the user might get a better/more concise hint.

Reply:
Oh, I missed the `set version` above. So it's
```suggestion
  version: {{ version }}
```
then.

Reply:
They should get an empty string. but yeah, probably good to get the version from above